python/modules/Stop0lObjectsProducer.py

```diff
@@ -133,21 +133,23 @@ def SelSoftb(self, isv):
 
 
     def SelJets(self, jet):
-        if jet.pt < 20 or math.fabs(jet.eta) > 2.4 :
+        if jet.pt < 20 or math.fabs(jet.eta) > 2.4:
             return False
         return True
 
     def SelPhotons(self, photon):
-        if photon.pt < 200:
+        if photon.pt < 220:
```

Comment:
This pt cut should be removed , as TTZ might use photon for ttG study.

Reply:
Ah, and TTZ doesn't need the photon trigger... ok.

Reply:
I have removed the pt cut now.